src/runtime/go-test.sh

```diff
@@ -15,14 +15,19 @@ long_options=(
 	[package:]="Specify test package to run"
 )
 
-# Set default test run timeout value.
-#
-# KATA_GO_TEST_TIMEOUT can be set to any value accepted by
-# "go test -timeout X"
-timeout_value=${KATA_GO_TEST_TIMEOUT:-30s}
+# Set up go test flags
+go_test_flags=${KATA_GO_TEST_FLAGS}
+if [ -z "$go_test_flags" ]; then
+    # KATA_GO_TEST_TIMEOUT can be set to any value accepted by
+    # "go test -timeout X"
+    go_test_flags="-v -timeout ${KATA_GO_TEST_TIMEOUT:-30s}"
+
+    # -race flag is not supported on s390x
+    [ "$(go env GOARCH)" != "s390x" ] && go_test_flags="$go_test_flags -race"
```

Comment:
Right, I always forget exactly what's in base shell and what's just in bash, and habitually try to avoid bashisms.  That does make it clearer here, though, so I'll use this.